src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpgradeForm.spec.jsx

```diff
@@ -730,11 +731,12 @@ describe('UpgradeForm', () => {
     })
 
     describe('when the user chooses less than the number of active users', () => {
```

Comment:
nitpick: add a "when a customer is undergoing a trial"

Reply:
Fixed